Gemfile

```
@@ -6,15 +6,17 @@ gem "activesupport", require: false
 gem "mry", require: false
 gem "parser"
 gem "pry", require: false
-gem "rubocop", "0.92.0", require: false
-gem "rubocop-i18n", require: false
-gem "rubocop-migrations", require: false
+gem "rubocop", "1.4.1", require: false
+gem "rubocop-i18n", require: false,
+  github: "mvz/rubocop-i18n", ref: "5e824a47a88732830a8b60e46697bbb0a2795c19"
```

```suggestion
gem "rubocop-i18n", require: false
```
This change should be applied when https://github.com/puppetlabs/rubocop-i18n/pull/46 is merged and released.

---

@DanielWright wouldn't this PR be merged faster, if you stick to @mvz s [branch](https://github.com/mvz/rubocop-i18n/tree/upgrade-to-rubocop-1-0) for now?
I suggest to move this change into another pull request.